src/Filter/AbstractDateFilter.php

```diff
@@ -119,12 +123,36 @@ public function getRenderSettings()
         ]];
     }
 
+    /**
+     * @param string $field
+     * @param array  $data
+     */
+    protected function applyTypeIsLessEqual(ProxyQueryInterface $queryBuilder, $field, $data)
+    {
+    }
+
+    /**
+     * @param string $field
+     * @param array  $data
+     */
+    protected function applyTypeIsGreaterThan(ProxyQueryInterface $queryBuilder, $field, $data)
+    {
+    }
+
+    /**
+     * @param string $field
+     * @param array  $data
+     */
+    protected function applyTypeIsEqual(ProxyQueryInterface $queryBuilder, $field, $data)
```

Comment:
These methods are required since they are called in this abstract class, they were only implemented in the extending classes.

Reply:
:thinking: why? This class is marked as `abstract`, I don't see in what situation they will not be implemented…

Reply:
What I meant is that they are called from [within the abstract class](https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/blob/38bc4929d32979b57c592e72be7a2b77bfc0d06a/src/Filter/AbstractDateFilter.php#L67) and they weren't defined, what I did wrong is that they should be declared as `abstract` as well.